services/kuber/server/server.go

```diff
@@ -31,6 +32,20 @@ const (
 	outputDir        = "services/kuber/server/clusters"
 )
 
+type (
+	IPPair struct {
+		PublicIP  net.IP `json:"public_ip"`
+		PrivateIP net.IP `json:"private_ip"`
+	}
+
+	ClusterMetadata struct {
+		// NodeIps maps node-name to public-private ip pairs.
+		NodeIps map[string]IPPair `json:"node_ips"`
```

Comment:
Do you think we should add private IP to this metadata as well? I am not sure if we can get this info using kubectl.
@Despire @MarioUhrik

Reply:
Not sure what you mean @jaskeerat789  there already is a private/public IP https://github.com/Berops/claudie/blob/acbfaad00155d0ad0faac82604df90b7f5b3f222/services/kuber/server/server.go#L91-L95